server/sonar-db-dao/src/main/java/org/sonar/db/organization/OrganizationMemberDao.java

```diff
@@ -55,4 +58,21 @@ public void deleteByOrganizationUuid(DbSession dbSession, String organizationMem
     return mapper(dbSession).selectOrganizationUuidsByUser(userId);
   }
 
+  /**
+   *
+   * @param loginOrganizationConsumer {@link BiConsumer}<String,String> (login, organization uuid)
+   */
+  public void selectForUserIndexing(DbSession dbSession, List<String> logins, BiConsumer<String, String> loginOrganizationConsumer) {
+    executeLargeInputsWithoutOutput(logins, list -> mapper(dbSession).selectForIndexing(list)
+      .forEach(row -> loginOrganizationConsumer.accept(row.get("login"), row.get("organizationUuid"))));
```

Comment:
Interesting usage of BiConsumer ! I wasn't aware of this class, but it's good to know,